admin_pages/events/Events_Admin_Page.core.php

```diff
@@ -985,25 +985,24 @@ public function extra_permalink_field_buttons($return, $id, $new_title, $new_slu
     protected function _events_overview_list_table()
     {
         do_action('AHEE_log', __FILE__, __FUNCTION__, '');
-        $this->_template_args['after_list_table']                           =
-            ! empty($this->_template_args['after_list_table'])
-                ? (array) $this->_template_args['after_list_table']
-                : [];
-        $this->_template_args['after_list_table']['view_event_list_button'] = EEH_HTML::br()
-                                                                              . EEH_Template::get_button_or_link(
-                get_post_type_archive_link('espresso_events'),
-                esc_html__('View Event Archive Page', 'event_espresso'),
-                'button'
-            );
-        $this->_template_args['after_list_table']['legend']                 =
-            $this->_display_legend($this->_event_legend_items());
-        $this->_admin_page_title                                            .= ' '
-                                                                               . $this->get_action_link_or_button(
+        $after_list_table                           = [];
+        $after_list_table['view_event_list_button'] = EEH_HTML::br();
+        $after_list_table['view_event_list_button'] .= EEH_Template::get_button_or_link(
+            get_post_type_archive_link('espresso_events'),
+            esc_html__('View Event Archive Page', 'event_espresso'),
+            'button'
+        );
+        $after_list_table['legend'] = $this->_display_legend($this->_event_legend_items());
+        $this->_admin_page_title    .= ' ' . $this->get_action_link_or_button(
                 'create_new',
                 'add',
                 [],
                 'add-new-h2'
             );
+        $this->_template_args['after_list_table']   = array_merge(
+            $this->_template_args['after_list_table'],
+            $after_list_table
+        );
```

Comment:
`$this->_template_args['after_list_table']` is now an empty string, so array_merge() is throwing a warning on the event list table

Reply:
kk we can just type cast that value, ie:

```php
        $this->_template_args['after_list_table']   = array_merge(
            (array) $this->_template_args['after_list_table'],
            $after_list_table
        );
```